### PR TITLE
TestResults fixes

### PR DESCRIPTION
**Pull request by kaipot**
#### Rationale
This fixes various bugs in the TestResults module.

#### Changes
* Top leaks on rundown now includes handle leaks
* Remove "The Beginning of Time" range options
* Fix date discrepancies (e.g. with rundown graphs grouping runs by date differently than summary table)

### CODE REVIEW

File: testresults/src/org/labkey/testresults/view/RunDownBean.java [hunk 1]
```diff
@@ -35,81 +37,72 @@ public RunDownBean(RunDetail[] runs, User[] users, String viewType, Date startDa
...
             }
-        });
-        Calendar cal = Calendar.getInstance();
-        cal.setTime(selectedDate);
-        TestResultsController.setToEightAM(cal);
-        selectedDate = cal.getTime();
-        cal.add(Calendar.DATE, -1);
-        Date dateBefore1Day = cal.getTime();
+        }
+        return (double)totalMem / numMem;
```
Comment (vagisha) [inline testresults/src/org/labkey/testresults/view/RunDownBean.java]: Could we get a divide by zero error here?

File: testresults/src/org/labkey/testresults/view/RunDownBean.java [hunk 2]
```diff
@@ -35,81 +37,72 @@ public RunDownBean(RunDetail[] runs, User[] users, String viewType, Date startDa
...
             {
-                User u = getUserById(run.getUserid());
-                map.computeIfAbsent(u, k -> new ArrayList<>());
-                map.get(u).add(run);
+                numHandle++;
+                totalHandle += ((TestHandleLeakDetail) leak).getHandles();
             }
         }
-        return map;
+        return totalHandle / numHandle;
```
Comment (vagisha) [inline testresults/src/org/labkey/testresults/view/RunDownBean.java]: Divide by zero error possible?

**Review by vagisha — APPROVED** [review]
I left a couple of comments.

**Comment by kaipot** [comment]
> I left a couple of comments.

Divide by zero was definitely possible in those spots! Thanks for catching it.

**Pull request merged**